pkg/restapi/operation/operations.go

```diff
@@ -14,22 +14,24 @@ import (
 	"net/url"
 
 	"github.com/google/uuid"
+	"github.com/trustbloc/edge-core/pkg/storage"
 
 	support "github.com/trustbloc/hub-kms/pkg/internal/common"
 	"github.com/trustbloc/hub-kms/pkg/keystore"
-	"github.com/trustbloc/hub-kms/pkg/restapi/models"
 )
 
 const (
 	kmsBasePath            = "/kms"
 	keystoreEndpoint       = kmsBasePath + "/keystore"
 	createKeystoreEndpoint = kmsBasePath + "/createKeystore"
 
-	keystoreIDFormat = "urn:uuid:%s"
+	keystoreIDFormat  = "urn:uuid:%s"
+	keystoreConfigKey = "config"
 
-	readRequestFailure           = "Failed to read the request body: %s"
-	createKeystoreFailure        = "Failed to create a keystore: %s"
-	receivedInvalidConfiguration = "Received invalid keystore configuration: %s"
+	readRequestFailure              = "Failed to read the request body: %s"
+	createKeystoreFailure           = "Failed to create a keystore: %s"
+	receivedInvalidConfiguration    = "Received invalid keystore configuration: %s"
```

Comment:
use %w for wrapping errors

Reply:
these messages are sent back to the user in HTTP response in case of errors, so "%s" should be fine here